packages/react/src/components/cookieConsent/CookieConsent.stories.tsx

```diff
@@ -1,12 +1,8 @@
 import React, { useState } from 'react';
 
-import {
-  Category,
-  CookieData,
-  Provider as CookieContextProvider,
-  SupportedLanguage,
-  useCookieConsentContext,
-} from './CookieConsentContext';
+import { useConsentContext } from './contexts/ConsentContext';
```

Comment:
Since this useConsentContext is globally exported variable, I think it should probably include the word __cookie__ in it for enough context for the user